### DISCUSSION TITLE
Self hosting console errors api/profile/permissions 404

### DISCUSSION THREAD

**Discussion by rgfx** [supabase/supabase#8983] [Questions]
Getting these console errors doing self-hosting, do you know what am doing wrong. App seems to be working file. 

![image](https://user-images.githubusercontent.com/2388078/190278543-5885450d-6aa7-4a27-acdd-d3137cc006fb.png)

**Comment by secretgspot** [comment]
same issue reported here https://github.com/supabase/supabase/issues/8721

**Comment by secretgspot** [comment]
note to self and others that issue possibly being worked on in https://github.com/supabase/supabase/pull/8988

**Comment by anasfik** [comment]
hi, did you managed to fix the issue or find anything uselful about it ?

**Comment by nicetomytyuk** [comment]
Any update on this issue?

**Comment by crackedbeefcake** [comment]
some info would be valuable
